neuro_san/internals/run_context/langchain/llms/langchain_llm_factory.py

```diff
@@ -65,54 +65,17 @@ def create_base_chat_model(self, config: Dict[str, Any]) -> BaseLanguageModel:
         """
         raise NotImplementedError
 
-    def create_llm_resources_with_client(self, config: Dict[str, Any],
-                                         llm_client: LangChainLlmClient = None) -> LangChainLlmResources:
+    def create_llm_resources(self, config: Dict[str, Any]) -> LangChainLlmResources:
```

Comment:
Change the interface before going live with the first merge released.
There still might be more changes here. See comments in ClientPolicy above.